doc/dev/perfstress_tests.md

```diff
@@ -1,9 +1,11 @@
 # Table of Contents
```

Comment:
Can just add a markdown comment like `[comment]: # ( cspell:ignore perfstresstest batchperftest )`  at the end of the Table of Contents section to appease cspell.